core/src/main/scala/scalaz/std/Future.scala

```diff
@@ -5,7 +5,8 @@ import scala.concurrent.{Await, CanAwait, ExecutionContext, Future}
 import scala.concurrent.duration.Duration
 
 trait FutureInstances1 {
-  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with Cobind[Future] =
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Cobind[Future] with MonadError[({type λ[α,β] = Future[β]})#λ, Throwable] with Catchable[Future] =
```

Comment:
@S11001001 I tried this and no luck.  I can keep playing around, but any other suggestions are appreciated. :)